### DISCUSSION TITLE
Configure Otel span attributes for exporting to google cloudtrace Big Query table

### DISCUSSION THREAD

**Discussion by vinodkumarpandiyan** [open-telemetry/opentelemetry-collector-contrib#36665] [Q&A]
Hi team, 

Our applications generates traces through otel with more than 32 span attributes which are sent to Grafana and google cloudtrace. Since google cloud can keep only 32 span attributes, it fetches 32 attributes from the span randomly. 

I want to configure in Otel collector to export our desired span attributes to google cloud. I should able to exclude the span attributes which are not required. 

Could you tell me how can i configure the required span attributes which needs to be sent to google cloud?

Thanks for your assistance.

**Comment by vinodkumarpandiyan** [comment]
Hi Team, Any update on the above request?

**Reply by bianchidotdev** [reply]
I believe you can use the [redaction processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/redactionprocessor) now to accomplish this. One way would be to disable all attributes by default, and allow a list of 32 span attributes that you care about.

Example:
```yml
processors:
  redaction:
    allow_all_keys: false
    allowed_keys:
      - span_attribute_key_1
      - ...
```
